server/config/nl_page/chart_titles_by_sv.json

```diff
@@ -1325,47 +1325,47 @@
   },
   "Annual_Amount_Emissions_Oxo_Oxochromiooxy_Chromium_SCC_1_ExternalCombustion": {
     "blocklist": [],
-    "title": "Annual Emissions: Oxo Chromium, External Combustion"
+    "title": "Annual Emissions: Chromium Oxide, External Combustion"
   },
   "Annual_Amount_Emissions_Oxo_Oxochromiooxy_Chromium_SCC_21_StationarySourceFuelCombustion": {
     "blocklist": [],
-    "title": "Annual Emissions: Oxo Chromium, Stationary Source Fuel Combustion"
+    "title": "Annual Emissions: Chromium Oxide, Stationary Source Fuel Combustion"
   },
   "Annual_Amount_Emissions_Oxo_Oxochromiooxy_Chromium_SCC_22_MobileSources": {
     "blocklist": [],
-    "title": "Annual Emissions: Oxo Chromium, Mobile Sources"
+    "title": "Annual Emissions: Chromium Oxide, Mobile Sources"
   },
   "Annual_Amount_Emissions_Oxo_Oxochromiooxy_Chromium_SCC_23_IndustrialProcesses": {
     "blocklist": [],
-    "title": "Annual Emissions: Oxo Chromium, Industrial Processes"
+    "title": "Annual Emissions: Chromium Oxide, Industrial Processes"
   },
   "Annual_Amount_Emissions_Oxo_Oxochromiooxy_Chromium_SCC_24_SolventUtilization": {
     "blocklist": [],
-    "title": "Annual Emissions: Oxo Chromium, Solvent Utilization"
+    "title": "Annual Emissions: Chromium Oxide, Solvent Utilization"
   },
   "Annual_Amount_Emissions_Oxo_Oxochromiooxy_Chromium_SCC_26_WasteDisposalTreatmentAndRecovery": {
     "blocklist": [],
-    "title": "Annual Emissions: Oxo Chromium, Waste Treatment & Recovery"
+    "title": "Annual Emissions: Chromium Oxide, Waste Treatment & Recovery"
   },
   "Annual_Amount_Emissions_Oxo_Oxochromiooxy_Chromium_SCC_28_MiscellaneousAreaSources": {
     "blocklist": [],
-    "title": "Annual Emissions: Oxo Chromium, Misc Sources"
+    "title": "Annual Emissions: Chromium Oxide, Misc Sources"
   },
   "Annual_Amount_Emissions_Oxo_Oxochromiooxy_Chromium_SCC_2_InternalCombustionEngines": {
     "blocklist": [],
-    "title": "Annual Emissions: Oxo Chromium, Internal Combustion Engine"
+    "title": "Annual Emissions: Chromium Oxide, Internal Combustion Engine"
   },
   "Annual_Amount_Emissions_Oxo_Oxochromiooxy_Chromium_SCC_3_IndustrialProcesses": {
     "blocklist": [],
-    "title": "Annual Emissions: Oxo Chromium, Industrial Processes"
+    "title": "Annual Emissions: Chromium Oxide, Industrial Processes"
```

Comment:
Is Oxo Chromium and Chromium oxide the same thing? (I don't know the chemistry)

Reply:
I think the original Oxo(Oxochromioxy) Chromium refers to Cr2O3 (two chromiums, three oxygen atoms) which is called Chromium (III) oxide, but has many possible names depending on convention (see [its wikipedia entry](https://en.wikipedia.org/wiki/Chromium(III)_oxide)).

Chromium Oxide would be the class of CrXOY compounds which Chromium (III) oxide falls under. I couldn't find the original name used by the EPA (their data portal requires permissions to access) but they do have fact sheets about air emissions that refer to chromium compounds (https://www.epa.gov/sites/default/files/2015-07/documents/acrylic_etc_fr_fs061507.pdf and https://www.epa.gov/sites/default/files/2016-09/documents/chromium-compounds.pdf) so I've omitted the (III) from the SV name just in case.

As far as I'm aware, oxo chromium is not a valid name for any chemical.